v1.1.0/docs/01-why.mdx

```diff
@@ -1,21 +1,31 @@
 # Why?
 
-Edtr.io was born from the needs of the learning platform [serlo.org](https://serlo.org/)
-and with all the experience we gathered from creating educational resources by a diverse group of authors.
+Edtr.io was born from the needs of the learning platform
+[serlo.org](https://serlo.org/) and with all the experience we gathered from
+creating educational resources by a diverse group of authors.
 
-We tried out a lot of other editing libraries before we settled on creating our own.
-Existing rich text libraries (e.g. [Slate](https://www.slatejs.org/), [Draft.js](https://draftjs.org/))
-came closest to what we needed: a way to build intuitive WYSIWYG rich text editors.
+We tried out a lot of other editing libraries before we settled on creating our
+own (like [Slate](https://www.slatejs.org/) or
+[Draft.js](https://draftjs.org/)). However we noticed that the rich text part is
+only a small (albeit very important) foundation of what we need.
 
-After using them for a while, we noticed that the rich text part is only a small (albeit very important) foundation of what we needed, though.
-When creating interactive learning resources, we often need much more than that. Think for example
-about multiple choice exercises and more interactive learning games. Providing an editing experience
-for such use cases often requires us to have a tight control over the user experience. Trying to achieve that with something like Slate was hard, though:
+When creating interactive learning resources, we need a better suited
+abstraction for editing than the one a rich text library can offer. It's like
+the comparision between Assembly and a high level programming language: With
+Edtr.io you will have a better productivity since you do not need to deal with
+the low level details.
```

Comment:
```suggestion
When creating interactive learning resources, we need a better abstraction
for developing editing experiences that are not primarily dealing with rich text.
It's similar to comparing Assembly with a high-level programming language: With
Edtr.io you will have increased productivity since you do not have to deal with
low-level details in most cases.
```